projects/inverse/index.js

```diff
@@ -0,0 +1,205 @@
+      })),
+      abi: abi["getCash"],
+    })
+  ).output;
+}
+
+async function getTotalSupplies(block, tokens) {
+  return (
+    await sdk.api.abi.multiCall({
+      block,
+      calls: _.map(tokens, (token) => ({
+        target: token,
+      })),
+      abi: abi["totalSupply"],
+    })
+  ).output;
+}
+
+async function anchorTVL(block) {
+  const balances = {};
+
+  const tokens = await getAllTokens(block);
+  const [allUnderlying, cashes] = await Promise.all([
+    getAllUnderlying(block, tokens),
+    getCashes(block, tokens),
+  ]);
+
+  _.each(tokens, (token) => {
+    let cash = _.find(
+      cashes,
+      (result) => result.success && result.input.target === token
+    );
+    let underlying = _.find(
+      allUnderlying,
+      (result) => result.success && result.input.target === token
+    );
+    if (cash && underlying) {
+      balances[underlying.output] = BigNumber(
+        balances[underlying.output] || 0
+      ).plus(cash.output);
+    }
+  });
+
+  return balances;
+}
+
+async function vaultsTVL(block) {
+  const balances = {};
+
+  const [allUnderlying, totalSupplies] = await Promise.all([
+    getAllUnderlying(block, vaults),
+    getTotalSupplies(block, vaults),
+  ]);
+
+  _.each(vaults, (token) => {
+    let totalSupply = _.find(
+      totalSupplies,
+      (result) => result.success && result.input.target === token
+    );
+    let underlying = _.find(
+      allUnderlying,
+      (result) => result.success && result.input.target === token
+    );
+    if (totalSupply && underlying) {
+      balances[underlying.output] = BigNumber(
+        balances[underlying.output] || 0
+      ).plus(totalSupply.output);
+    }
+  });
+
+  return balances;
+}
+
+async function stabilizerTVL(block) {
+  const totalSupply = (
+    await sdk.api.abi.call({
+      block,
+      target: dola,
+      abi: abi["totalSupply"],
+    })
+  ).output;
+
+  return {
+    [dai]: BigNumber(totalSupply),
+  };
+}
+
+async function tvl(timestamp, block) {
+  const balances = {};
+
+  const [
+    anchorBalances,
+    vaultBalances,
+    stabilizerBalances,
+  ] = await Promise.all([
+    anchorTVL(block),
+    vaultsTVL(block),
+    stabilizerTVL(block),
+  ]);
+
+  _.each(_.pairs(anchorBalances), ([token, value]) => {
+    const balance = BigNumber(balances[token] || 0);
+    balances[token] = balance.plus(BigNumber(value)).toFixed();
+  });
+
+  _.each(_.pairs(vaultBalances), ([token, value]) => {
+    const balance = BigNumber(balances[token] || 0);
+    balances[token] = balance.plus(BigNumber(value)).toFixed();
+  });
+
+  _.each(_.pairs(stabilizerBalances), ([token, value]) => {
+    const balance = BigNumber(balances[token] || 0);
+    balances[token] = balance.plus(BigNumber(value)).toFixed();
+  });
+
+  return balances;
+}
+
+/*==================================================
+  Exports
+  ==================================================*/
+
+module.exports = {
+  name: "Inverse",
+  token: "INV",
+  category: "lending",
```

Comment:
Are we "lending" or "assets"? I'm leaning towards "assets". Probably better answered by @nourharidy.

Reply:
It should be assets as we're not purely lending, by looking at the website assets is like a wildcard (see badger).

Reply:
Agreed about assets.

Reply:
Yeah I was leaning between the two. Will change to assets.